tests/unit/test_column_noise.py

```diff
@@ -555,6 +558,51 @@ def test_write_wrong_digits(dummy_dataset):
             assert (noised_data[ssn].str[i].str.isdigit()).all()
 
 
+def test_write_wrong_digits(dummy_dataset):
+    # This is a quicker (less robust) version of the test above to run on push and pull
```

Comment:
I'd remove the bit about running on push and pull requests. The test is not tied only to CI (e.g. I can run from my terminal)